stdlib/socketserver.pyi

```diff
@@ -53,14 +54,14 @@ class BaseServer:
     def serve_forever(self, poll_interval: float = ...) -> None: ...
     def shutdown(self) -> None: ...
     def server_close(self) -> None: ...
-    def finish_request(self, request: _RequestType, client_address: _AddressType) -> None: ...
+    def finish_request(self, request: _RequestType, client_address: _RetAddress) -> None: ...
```

Comment:
Should we also use the `_RetAddress` alias for the second parameter of `RequestHandlerClass` on lines 45, 49, and 51? (A few other places in this file as well.)

Reply:
Yes, good point